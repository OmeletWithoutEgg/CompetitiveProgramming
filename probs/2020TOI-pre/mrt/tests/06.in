68
97 30 90 78 18 30 65
71 11 48 20 23 60 4
5 4 0 1 0 4 0
41 25 14 1 32 15 19
37 14 28 0 19 19 30
79 27 4 1 43 14 60
7 3 2 0 4 4 6
13 0 2 2 7 3 4
71 11 1 46 54 9 30
43 37 39 32 35 15 20
43 2 18 41 7 24 1
37 0 11 32 21 12 0
53 37 3 23 47 44 1
2 0 1 0 1 1 0
31 0 6 28 14 26 6
17 6 1 6 10 13 7
47 38 46 5 19 36 41
53 8 7 13 20 1 4
47 45 25 4 38 22 17
31 11 5 12 25 6 15
17 10 6 1 0 10 1
31 25 6 6 0 7 20
73 55 17 69 5 66 47
41 30 9 31 29 17 26
41 38 35 30 7 17 31
97 10 93 57 95 4 87
2 1 1 0 1 1 1
73 42 36 39 29 5 58
67 44 20 29 15 46 32
23 15 2 0 18 17 8
83 39 70 58 53 14 66
59 13 36 34 55 5 52
19 3 7 15 8 12 2
89 73 81 26 8 28 74
71 32 7 32 28 38 17
37 11 21 13 20 24 30
97 49 46 87 37 38 61
89 2 47 33 70 39 9
67 64 28 51 0 59 64
37 13 26 7 4 10 3
19 2 7 6 8 0 11
47 11 14 18 34 30 32
5 1 2 4 0 1 2
2 1 1 0 1 1 1
31 25 7 13 24 4 4
67 51 63 32 5 28 44
83 78 80 9 10 50 74
13 0 5 4 2 10 11
61 57 39 31 11 25 58
89 38 12 38 13 45 38
67 8 59 59 32 54 6
47 44 7 26 31 17 13
3 1 0 0 0 2 1
71 65 54 66 39 30 65
5 1 1 2 1 3 4
97 24 33 27 73 43 81
37 18 0 30 7 10 12
11 3 1 5 8 0 1
5 3 0 3 4 4 1
19 15 9 5 0 1 14
67 50 45 38 1 39 64
5 4 1 0 4 4 1
89 16 16 3 64 74 53
29 9 9 4 14 9 6
11 3 0 1 9 6 6
23 4 19 20 16 2 14
29 4 2 7 24 12 26
19 10 1 15 12 6 12
