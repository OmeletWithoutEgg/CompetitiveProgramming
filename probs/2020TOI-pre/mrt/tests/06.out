95 64
36 58
0 0
29 6
21 0
40 15
5 3
4 1
59 49
31 24
18 9
4 36
45 19
0 1
24 29
2 2
38 3
29 6
8 2
2 29
10 1
15 25
61 44
33 6
19 21
20 40
1 1
38 32
60 46
19 19
48 22
56 41
15 8
56 19
30 8
10 24
45 16
19 64
21 28
16 1
18 2
16 3
1 2
1 1
22 13
56 39
57 8
11 6
24 15
46 67
40 27
8 42
2 1
52 57
4 2
3 61
4 32
4 0
2 2
11 17
6 62
0 4
11 59
0 3
7 1
3 9
9 19
17 12
